Bixal/move.mil
# Feature 47 theme pro tip pgraph

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

provides a template to display pro-tips
allow the user to assign where that pro-tip is displayed
provides a new icon--pro-tip.svg in the asset
allows the npm build script to glob through all /image directories

## Testing

To verify the changes proposed in this pull request…

1. Pull these changes locally
2. npm build
3. create a new basic page content
4. give it a title
5. add the Pro-tip Paragraph

**Test A**

- give it a title
- set alignment to top
- include markup in the WYSIWYG editor

<strong>Pro-Tip:</strong><p>Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec velit neque, auctor sit amet aliquam vel, ullamcorper sit amet ligula. Praesent sapien massa, convallis a pellentesque nec, egestas non nisi. Sed porttitor lectus nibh. Nulla quis lorem ut libero malesuada feugiat. Donec rutrum congue leo eget malesuada. Sed porttitor lectus nibh. Praesent sapien massa, convallis a pellentesque nec, egestas non nisi. Proin eget tortor risus. Donec rutrum congue leo eget malesuada.</p>

**Test B**

- you can try left and right alignments but will need to add another basic text paragraph for it to make sense

## Screenshots

**mobile**

![screencapture-move-mil-localhost-8000-node-2-2018-04-04-13_59_30](https://user-images.githubusercontent.com/37077057/38326443-d82a408c-3813-11e8-9469-9f5b1d91ca25.png)

**desktop**
![screencapture-move-mil-localhost-8000-node-2-2018-04-04-13_59_04](https://user-images.githubusercontent.com/37077057/38326470-e1f1377e-3813-11e8-9f75-86b161a4b9e1.png)
